replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicationSiteImpl.java

```diff
@@ -132,15 +132,15 @@ public boolean isRemoteManaged() {
   }
 
   @Override
-  public void fromMap(Map<String, Object> properties) {
-    super.fromMap(properties);
-    final int version = super.getVersion();
+  public int fromMap(Map<String, Object> properties) {
+    int propVersion = super.fromMap(properties);
```

Comment:
✏️ How about renaming it `currentVersion` or `serializedVersion`?

Reply:
Changed to serializedVersion